dash/testing/wait.py

```diff
@@ -0,0 +1,91 @@
+# pylint: disable=too-few-public-methods
+"""Utils methods for pytest-dash such wait_for wrappers"""
+import time
+import logging
+from selenium.common.exceptions import WebDriverException
+from dash.testing.errors import TestingTimeoutError
+
+
+logger = logging.getLogger(__name__)
+
+
+def until(
+    wait_cond,
+    timeout,
+    poll=0.1,
+    msg="expected condition not met within timeout",
+):  # noqa: C0330
+    res = None
```

Comment:
minor, but do we want to init with a `wait_cond()` so if it's already satisfied we don't even need to wait one `poll` period?